profile/README.md

```diff
@@ -1,8 +1,79 @@
+- [Eclipse Foundation](https://www.eclipse.org)
+- [Eclipse Foundation Project handbook](https://www.eclipse.org/projects/handbook/)
+- [Catena-X Automotive Network e.V.](https://catena-x.net)
+- [Catena-X - Library](https://catenax-ev.github.io/)
+- [Eclipse Tractus-x Project Page](https://projects.eclipse.org/projects/automotive.tractusx)
+
+## Stay Connected
+
+- Join our mailing list: [subscribe here](https://accounts.eclipse.org/mailing-list/tractusx-dev)
+- Follow us on our [webpage](https://eclipse-tractusx.github.io/) and our [news sectcion](https://eclipse-tractusx.github.io/blog)
+- Join our matrix chats at: [Tractus-X](https://matrix.to/#/#automotive.tractusx:matrix.eclipse.org)
+
+## Our Community
+
+Meet the project leads of Eclipse Tractus-X:
+
+| ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/stephanbcbauer?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/matbmoser?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/evegufy?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Daniel Miehle](https://images.weserv.nl/?url=avatars.githubusercontent.com/danielmiehle?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Björn Roy](https://images.weserv.nl/?url=avatars.githubusercontent.com/giterrific?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d)  |
+|:--------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------:|
+|                                     **[Stephan Bauer](https://github.com/stephanbcbauer)**                                      | **[Mathias Brunkow Moser](https://github.com/matbmoser)**           |                                                 **[Evelyn Gurschler](https://github.com/evegufy)**                                                 |                                                **[Daniel Miehle](https://github.com/danielmiehle)**                                                 |                                               **[Björn Roy](https://github.com/giterrific)**                                               |
+
+Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) of Eclipse Tractus-X.
```

Comment:
Should not the committers come first?

Reply:
You mean before the contributors? Or Contributor and Committer before Project lead?

Reply:
done